File: easy/3986_number_of_elapsed_seconds_between_two_times/solution.py
class Solution:
    def secondsBetweenTimes(self, startTime: str, endTime: str) -> int:
        startHours = int(startTime[:2])
        startMinutes = int(startTime[3:5])
        startSeconds = int(startTime[6:])

        endHours = int(endTime[:2])
        endMinutes = int(endTime[3:5])
        endSeconds = int(endTime[6:])

        return (
            3600 * (endHours - startHours)
            + 60 * (endMinutes - startMinutes)
            + (endSeconds - startSeconds)
        )

File: easy/3986_number_of_elapsed_seconds_between_two_times/solution.cpp
#include <string>

class Solution {
public:
  int secondsBetweenTimes(std::string startTime, std::string endTime) {
    int startHours, startMinutes, startSeconds;
    sscanf(startTime.c_str(), "%d:%d:%d", &startHours, &startMinutes,
           &startSeconds);

    int endHours, endMinutes, endSeconds;
    sscanf(endTime.c_str(), "%d:%d:%d", &startHours, &startMinutes,
           &startSeconds);

    return (3600 * (endHours - startHours) + 60 * (endMinutes - startMinutes) +
            (endSeconds - startSeconds));
  }
};
